dl/src/main/scala/com/intel/analytics/bigdl/optim/DistriOptimizer.scala

```diff
@@ -571,6 +614,25 @@ class DistriOptimizer[T: ClassTag] (
 
     model
   }
+
+  private def getLatestFile(fileName: String): String = {
+    val fl = new java.io.File(checkpointPath.get)
+    val files = fl.listFiles(new FilenameFilter {
+      override def accept(dir: File, name: String): Boolean = {
+        name.startsWith(fileName)
+      }
+    })
+
+    var lastMod = Long.MinValue
```

Comment:
what about start training time? It's should not be conflict and more meaningful than UUID